test/integration/default/controls/security_test.rb

```diff
@@ -11,9 +11,17 @@
     it { should exist }
   end
 
-  describe command('/usr/bin/security find-certificate /Users/vagrant/Library/Keychains/login.keychain') do
+  describe file('/Users/vagrant/foobar.cer') do
+    it { should exist }
+  end
+
+  describe command('/usr/bin/security find-certificate /Users/vagrant/Library/Keychains/test.keychain') do
     its('stdout') { should include 'Test' }
   end
+
+  describe command('/usr/bin/security find-certificate /Users/vagrant/Library/Keychains/login.keychain') do
```

Comment:
I think the output of this command only lists the first certificate found. Maybe it would be better to use `-n` (query by name) or `-e` (query by email) to be more specific. Even better would be to use `-p` to print the contents in PEM format and verify the contents exactly match those of the original PEM file that we installed before.

Reply:
The PEM is generated during the test recipe execution, so we can't hardcode it into the integration test. I've added the arguments to query all certificates in the keychain.